corehq/apps/groups/models.py

```diff
@@ -138,6 +138,7 @@ def set_user_ids(self, user_ids):
             self.remove_user(user_id)  # no option to save
 
         assert set(self.users) == target_users
+        return users_to_add, users_to_remove
```

Comment:
How about adding an assert to `test_set_user_ids` that checks the return value?

Reply:
Absolutely yes, thank you.

https://github.com/dimagi/commcare-hq/pull/30117/commits/a709afe509c8b1ad66c5c010ba4a60fdeed448e9